os/kernel/log_dump/log_dump.c

```diff
@@ -47,11 +49,13 @@
 #define MAX(a, b)		   (((a) > (b)) ? (a) : (b))
 #endif
 
-#define LOG_DUMP_OK			 0
+#define LOG_DUMP_OK			0	/* Used for successful return check */
+#define LOG_DUMP_OK_HEAPCHK		1	/* To check heap available after success return */
```

Comment:
Could you let me know the usages of this return value? Do we need to distinguish OK and OK_HEAPCHK?